<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>621</width>
    <height>446</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Device Registration</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../Downloads/digital_art_wallpaper_highway_3840x2160_67744_mm_90_7VZ_icon.ico</normaloff>../../Downloads/digital_art_wallpaper_highway_3840x2160_67744_mm_90_7VZ_icon.ico</iconset>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>260</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::WheelFocus</enum>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="text">
     <string>Register</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="username_textbox">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>60</y>
      <width>221</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Verdana&quot;;</string>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhLowercaseOnly</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="mac_textbox">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>110</y>
      <width>221</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Verdana&quot;;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="device_textbox">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>160</y>
      <width>221</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Verdana&quot;;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="sponsor_textbox">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>210</y>
      <width>221</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Verdana&quot;;</string>
    </property>
   </widget>
   <widget class="QLabel" name="progress_label">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>310</y>
      <width>381</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Verdana&quot;;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;Ready&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="change_mode">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>25</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="username_label">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>50</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 16pt &quot;Verdana&quot;;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ff0000;&quot;&gt;*&lt;/span&gt;Username&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="buddy">
     <cstring>username_textbox</cstring>
    </property>
   </widget>
   <widget class="QLabel" name="mac_address_label">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>100</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 16pt &quot;Verdana&quot;;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ff0000;&quot;&gt;*&lt;/span&gt;Mac Address&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="buddy">
     <cstring>mac_textbox</cstring>
    </property>
   </widget>
   <widget class="QLabel" name="device_label">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>150</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 16pt &quot;Verdana&quot;;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Device type&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="buddy">
     <cstring>device_textbox</cstring>
    </property>
   </widget>
   <widget class="QLabel" name="sponsor_label">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>200</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 16pt &quot;Verdana&quot;;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Your name&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="buddy">
     <cstring>sponsor_textbox</cstring>
    </property>
   </widget>
   <widget class="QLabel" name="instruction_label">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>0</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 8pt &quot;Verdana&quot;;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:11pt; color:#ff0000;&quot;&gt;*&lt;/span&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt; = Required&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <zorder>username_textbox</zorder>
   <zorder>pushButton</zorder>
   <zorder>mac_textbox</zorder>
   <zorder>device_textbox</zorder>
   <zorder>sponsor_textbox</zorder>
   <zorder>progress_label</zorder>
   <zorder>change_mode</zorder>
   <zorder>username_label</zorder>
   <zorder>mac_address_label</zorder>
   <zorder>device_label</zorder>
   <zorder>sponsor_label</zorder>
   <zorder>instruction_label</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>621</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="actionAdd_user_using_website"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="separator"/>
    <addaction name="actionHelp"/>
    <addaction name="separator"/>
   </widget>
   <addaction name="menuMenu"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAdd_user_using_website">
   <property name="text">
    <string>Add user using website</string>
   </property>
  </action>
  <action name="actionAdd_device_for_user_on_website">
   <property name="text">
    <string>Add device for user on website</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="text">
    <string>Help me please</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>username_textbox</tabstop>
  <tabstop>mac_textbox</tabstop>
  <tabstop>device_textbox</tabstop>
  <tabstop>sponsor_textbox</tabstop>
  <tabstop>pushButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>username_textbox</sender>
   <signal>returnPressed()</signal>
   <receiver>pushButton</receiver>
   <slot>animateClick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>312</x>
     <y>101</y>
    </hint>
    <hint type="destinationlabel">
     <x>297</x>
     <y>321</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mac_textbox</sender>
   <signal>returnPressed()</signal>
   <receiver>pushButton</receiver>
   <slot>animateClick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>331</x>
     <y>151</y>
    </hint>
    <hint type="destinationlabel">
     <x>278</x>
     <y>321</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>device_textbox</sender>
   <signal>returnPressed()</signal>
   <receiver>pushButton</receiver>
   <slot>animateClick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>393</x>
     <y>201</y>
    </hint>
    <hint type="destinationlabel">
     <x>243</x>
     <y>321</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sponsor_textbox</sender>
   <signal>returnPressed()</signal>
   <receiver>pushButton</receiver>
   <slot>animateClick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>421</x>
     <y>251</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>321</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
